### PR TITLE
PostBanner

### LINKED ISSUE #68: Post View Banner
## Description 
As a Po I want a user to be able to see a post on a separate page.

 **Describe the solution you'd like**
Create a Post View Banner 

 **Describe alternatives you've considered**
Does not apply

 **Additional context**
Does not apply

 **D.O.D.**
Profile Page banner should be according to design.

### PR DESCRIPTION
**Pull request by MiguelFieira**
closes #68 

 **What is it supposed to do**
If click on a post it should open a post view page with a banner according to design.

 **How can we test your branch**
- Go into branch ``68-post-view`` make sure to ``npm ci`` and click on a post on the homepage.
- Check of the banner is according to design

 **What browser have been checked**
- [X] Chrome
- [x] Firefox
- [x] Opera
- [ ] Internet Explorer
- [x] Edge
- [ ] Safari

### CODE REVIEW

**Review by Samvdg — APPROVED** [review]

File: src/pages/post/index.js [hunk 1]
```diff
@@ -1,18 +1,27 @@
...
+import { PostBanner } from '../../components'
 
+@inject('store') @observer
 class Post extends App {
   render() {
     const { postUrl } = this.props.match.params
     const pageNumber = postUrl ? Number(postUrl) + 1 : 1
 
     return (
       <>
```
Comment (vQFire) [inline src/pages/post/index.js]: You can just return `<Strandard />` no need to use `<>`

**Review by vQFire — CHANGES_REQUESTED** [review]
Just this one thing

File: src/pages/post/index.js (hunk 1 above)
Comment (MiguelFieira) [reply src/pages/post/index.js]: Fixed! Please recheck @vQFire

Comment (vQFire) [reply src/pages/post/index.js]: Fixed

**Review by vQFire — APPROVED** [review]
👍

**Review by Samvdg — APPROVED** [review]

**Comment by AjobK** [comment]
image

Very strange look for mobile. The name and role are not in the middle.

Maybe center the photo and info on mobile. (mobile-all-portrait media query)

**Comment by AjobK** [comment]
Also. the level is cut off by profile picture...

**Comment by AjobK** [comment]
When name or role is longer, the info goes underneath in a really strange way

**Review by AjobK — CHANGES_REQUESTED** [review]
Check comments

**Comment by AjobK** [comment]
Cursor pointer when I hover over the black box?

**Review by AjobK — CHANGES_REQUESTED** [review]
**1. Not centered.**
**2. The role should not be that wide.**
**3. The level is awkwardly inside profile picture**
image

**Why does this entire box have cursor pointer? Only profile picture and name**
image

**Comment by AjobK** [comment]
The role width is as wide as the users name. So if the users name is only 1 letter, we have a major problem.

**Comment by AjobK** [comment]
image
When screen is too small and name is longer..? Should be ellipsis

Example:
Emily washi...

**Comment by MiguelFieira** [comment]
@AjobK All issues mentioned have been fixed. Could you do a review again please?
Mention me if i missed something.

Thanks in advance!

Kind Regards,

[Miguel Fieira](http://www.linkedin.com/in/miguel-fieira/)
Developer [Seaqull](http://www.seaqull.com/)

**Review by AjobK — CHANGES_REQUESTED** [review]
Told u in discord. Ask again if needed.

**Pull request closed without merging**